MVP-UI/MVP-UI/AutorizationModule/Presenter/AutorizationPresenter.swift

```diff
@@ -42,11 +46,13 @@ extension AutorizationPresenter: AutorizationProtocol {
         guard let login = login else { return }
         guard let password = password else { return }
         if password.count < 6 {
-            view?.showSpashScreenOn()
-            view?.setTitleColorPassword(color: "splashColor", isValidatePassword: false)
-            DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
-                self.view?.showSpashScreenOff()
-            }
+            goToMainTabBarScreen()
+            // Здесь идет проверка пароля на валидность
+//            view?.showSpashScreenOn()
+//            view?.setTitleColorPassword(color: "splashColor", isValidatePassword: false)
+//            DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
+//                self.view?.showSpashScreenOff()
+//            }
```

Comment:
```suggestion
            // Здесь идет проверка пароля на валидность
//            view?.showSpashScreenOn()
//            view?.setTitleColorPassword(color: "splashColor", isValidatePassword: false)
//            DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
//                self.view?.showSpashScreenOff()
//            }
```
а куда делась проверка и почему закоменчено?